config/environments/test.rb

```diff
@@ -41,6 +41,12 @@
   # config.active_record.mass_assignment_sanitizer = :strict
 
   config.action_mailer.default_url_options = { :host => "localhost", :port => 3000 }
+
+  config.middleware.use ExceptionNotification::Rack,
+    email: {
+      sender_address: "exception@test.com",
+      exception_recipients: ["fake_developer@test.com"]
+    }
```

Comment:
Sure, I'll use `example.com` because it seems like `test.com` is used for DNS testing.